src/pages/HomePage.tsx

```diff
@@ -27,6 +27,7 @@ const HomePage = ({ streamersById, featuredStreamers, latestVideos, latestVideos
 
     return (
         <div className="home__page">
+            <h1>New: all videos have been indexed for your viewing pleasure. Enjoy!</h1>
```

Comment:
probably should be something else h1 which identify that its the home page
also **just my opinion** should not include pleaded or feelings to look professional